android/app/src/main/java/com/zulipmobile/notifications/NotifyReact.kt

```diff
@@ -17,6 +16,26 @@ import com.zulipmobile.emitEvent
  * TODO: Replace this with a fresh implementation based on RN upstream docs.
  */
 
+/**
+ * Recognize if an ACTION_VIEW intent came from tapping a notification; handle it if so
+ *
+ * Sends a 'notificationOpened' event to the JavaScript layer.
+ */
+internal fun maybeHandleViewIntent(intent: Intent, maybeReactContext: ReactContext?): Boolean {
```

Comment:
A key piece of this function's interface which the doc comment should mention is that it returns true just if it did handle the intent. (Just like the comment on `maybeHandleIntent`.)